### DISCUSSION TITLE
arm 32bit install error

### DISCUSSION THREAD

**Discussion by funny1023** [rustdesk/rustdesk#10043] [General]
### Bug Description

Using 32-bit old devices, different errors occurred when installing rustdesk-1.3.3-armv7-sciter.deb and rustdesk-1.1.9-raspberry-armhf.deb, which cannot be used.

### How to Reproduce

Reinstall after installation

### Expected Behavior

It can be used normally under the old version of arm 32 or compiled and used locally on the old version.

### Operating system(s) on local (controlling) side and remote (controlled) side

xubuntu 16.04

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.19 1.3.3

### Screenshots

Installation: rustdesk-1.1.9-raspberry-armhf.deb startup error
rustdesk: /lib/arm-linux-gnueabihf/libm.so.6: version `GLIBC_2.27' not found (required by rustdesk)
rustdesk: /lib/arm-linux-gnueabihf/libm.so.6: version `GLIBC_2.29' not found (required by rustdesk)
rustdesk: /lib/arm-linux-gnueabihf/libc.so.6: version `GLIBC_2.29' not found (required by rustdesk)
rustdesk: /lib/arm-linux-gnueabihf/libc.so.6: version `GLIBC_2.25' not found (required by rustdesk)
rustdesk: /lib/arm-linux-gnueabihf/libc.so.6: version `GLIBC_2.28' not found (required by rustdesk)
Error during installation when installing rustdesk-1.3.3-armv7-sciter.deb
Reading package list... Done
Analyzing a package's dependency tree
Reading status information... Complete
Note that rustdesk is selected instead of ./rustdesk-1.3.3-armv7-sciter.deb
Some packages cannot be installed. If you are using the unstable distribution, this may be
It's because the system cannot reach the state you requested. There may be some software you need in this version
The packages have not been created yet or they have been moved out of the Incoming directory.
The following information may be helpful in resolving the issue:
The following packages have unmet dependencies:
rustdesk: Depends on: libva-drm2 but cannot install it
Dependencies: libva-x11-2 but cannot install it
Dependencies: gstreamer1.0-pipewire but cannot install it
Recommended: libayatana-appindicator3-1 but can't install it
E: The error cannot be fixed because you required some packages to remain as they are, which breaks inter-package dependencies.

**Comment by funny1023** [comment]
这个问题可以解决吗

**Reply by xlionjuan** [reply]
Ubuntu 16 is EOL, try at least Ubuntu 20
